src/NosCore.Packets/ServerPackets/CharacterSelectionScreen/ClistPacket.cs

```diff
@@ -75,6 +75,6 @@ public class ClistPacket : PacketBase
         public int Design { get; set; }
 
         [PacketIndex(16)]
```

Comment:
I think it’s a bool not a byte

Reply:
Yep, you are right
0 : no rename window
1 : rename window